src/login.html

```diff
@@ -6,14 +6,17 @@
 
             <h1 style="margin-top:1em;text-align: left;">${HeaderPleaseSignIn}</h1>
 
-            <div style="height:0; overflow: hidden;"><input type="text" name="fakeusernameremembered" tabindex="-1" /><input type="password" name="fakepasswordremembered" tabindex="-1" /></div>
+            <div style="height:0; overflow: hidden;"><input type="text" name="fakeusernameremembered"
+                    tabindex="-1" /><input type="password" name="fakepasswordremembered" tabindex="-1" /></div>
```

Comment:
Actually, this change has particularly odd indentation. I would nest both input elements inside the div element with newlines, and place the closing tag for the div on its own line as well.

Reply:
This one I agree should definitely be changed to match nesting. To match previous style, I'll change the other lines to remove the newlines as well. They were added by my text editor formatting as those lines were over 120 chars long.